core/boards/nrf52840_pca10059-sub-slots-debug.dts

```diff
@@ -29,9 +29,11 @@
 		};
 
 		/*
-		 * Using last 2 pages of storage partition
+		 * Using 4 pages of scratch partition for OpenThread storage.
```

Comment:
I suggest you specify which pages the scratch is using.

Reply:
Done